src/kv/RocksDBStore.cc

```diff
@@ -491,6 +494,11 @@ int RocksDBStore::submit_transaction(KeyValueDB::Transaction t)
 
 int RocksDBStore::submit_transaction_sync(KeyValueDB::Transaction t)
 {
+  RocksDBTransactionImpl * _t =
```

Comment:
Agreed.  It is pretty easy/common for kv_sync_thread to have an empty transaction, in fact.. if there are no deferred ios and we're not bumping the max then this txn will be empty, but we're just using it as an fsync() basically.